tools/arriba-fusion-calling/2.0.0/arriba-fusion-calling__2.0.0.cwl

```diff
@@ -83,7 +83,7 @@ inputs:
     doc: |
       Output file with fusions that have passed all filters
     type: string?
-    default: fusion-passed.tsv
+    default: fusion.tsv
```

Comment:
Just wondering if this was an initial bug or you've upgraded to a new version - and as such there's no change to the DockerRequirement. If it was just the former (an initial bug) feel free to dismiss this comment

Reply:
Not really a bug! We have decided to update this value to be more consistent with our current batch setup.

Reply:
Ah apologies I got confused. I thought this might be a glob for an output file that had changed, rather than the name of an input file.